packages/@dcl/inspector/src/components/ImportAsset/utils.ts

```diff
@@ -175,12 +176,14 @@ async function processFile(file: File): Promise<BaseAsset> {
     return { blob: file, name, extension, error: extensionError }
   }
 
+  const isEmoteFile = await isEmote(file)
+
   const sizeError = validateFileSize(file.size)
   if (sizeError) {
-    return { blob: file, name, extension, error: sizeError }
+    return { blob: file, name, extension, error: sizeError, isEmote: isEmoteFile }
   }
 
-  return { blob: file, name, extension }
+  return { blob: file, name, extension, isEmote: isEmoteFile }
```

Comment:
I think declaring if the asset `isEmote` isn't something the import-asset flow should care about, since this should only care about files types and not differentiation between usage of the files. 
Maybe we could move the check of `isEmote` to the `Slider` component, since that's where we want to differentiate between usage of the same file type